README.md

```diff
@@ -54,14 +54,24 @@ Then you can build it:
 cargo build
 ```
 
-### Running
+### Configuring & Running
 
 You can check the needed cli parameters with the `-h` option:
 ```
 cargo run --release -- -h
 ```
 
-Note: an example configuration file can be seen [here](example_configuration_file.yml).
+In particular, the application expects a config file from you, to define which datasets (GTFS + GTFS RT) should be handled.
+
+This can be configured via a local file, or a HTTP url (see `-h` output for exact details).
+
+An example configuration file can be found in [here](example_configuration_file.yml), which you can use like this (in debug mode):
+
+```
+cargo run -- -c example_configuration_file.yml
```

Comment:
alternatively you can use

```
cargo run -- --gtfs <path or url of gtfs> --url <url fo the gtfs rt>
```
but I'm not sure it's worth mentionning since the config file is easier to use.

Reply:
Thanks, I've added that note!